
<h2>Description</h2>
<p>Mappability tracks</p>

<p>The minimum unique k-mer length tracks allow to check mappability of short or long reads from any individual, given how unique it is in the CHM13 assembly. The longer the k-mer length, the more mappable (unique) it is. Mappability from unique (single-copy) k-mers in CHM13 shows region inferred to be present as single copy (hetero + homo as CHM13 is nearly haploid) within CHM13, which might be different in other individuals.</p>

<h2>Description</h2>
<p>These tracks contain the minimum unique k-mer lengths, left- and right-anchored</p>

<h2>Methods</h2>
<p>All chromosome fasta sequences were concatenated. This concatenated sequence was then reverse complimented and appended to the original concatenated sequence. A suffix array was created from the combined sequence. Then for each position in the suffix array, the minimum unique k-mer length was found by taking the maximum longest common prefix value with the previous and next suffices and adding one. If the suffix originated from the forward orientation sequnce, it was assigned to the corresponding genomic position for the minimum unique k-mer length, left-anchored (MUL). Otherwise, the value was assigned to the corresponding genomic position for the minimum unique k-mer length, right-anchored (MUR). All positions whose MUL or MUR overlapped either 'N' bases or the start or end of a chromosome were marked as invalid in their respective tracks.</p>

<h2>Release history</h2>
<ol>
  <li>2021/05/20 released for CHM13v1.0
  <li>2022/03/26 released for CHM13v2.0
</ol>

<h2>Contacts</h2>
<ul>
  <li><a href="mailto:msauria@jhu.edu">Michael Sauria &lt;msauria@jhu.edu&gt;</a>
</ul>


<h2>Description</h2>
<p>Mappability from unique (single-copy) k-mers in the CHM13 assembly</p>

<h2>Methods</h2>
<p>Unique k-mers were generated by first creating an initial Illumina marker database produced with meryl (v1.0~v1.3) from Illumina PCRfree CHM13 reads obtained from the SRA (PRJNA269593). A k=21 and k=51 sizes were selected for database construction and kmers within single-copy multiplicity (greater than 42 and less than 133 for k=21, greater than 28 and less than 107 for k=51) were retained. Kmers outside of these ranges were considered erroneous or multi-copy. Utilising this initial singles database, assembly specific single copy marker databases were constructed by intersected unique kmers found in the assembly.</p>

<h2>Track Descriptions</h2>
<ol>
	<li>Density: Density of the overlapping marker kmers, with the maximum overlapping number of marker kmers being the size of the k. The more dense the markers are present, the region is highly unique and easily mappable.</li>
	<li>Marker Desert: Regions at least 50kbp away from the nearest marker.</li>
</ol>

<h2>Data Access</h2>
Meryl marker databases are available to download <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/alignments/marker/">here.</a>

<h2>Release history</h2>
<ol>
  <li>v1.0 </li>
  <li>v1.1 </li>
</ol>

<h2>Contacts</h2>
<ul>
  <li><a href="mailto:ann.mccartney2@mail.dcu.ie">Ann Mc Cartney</a>, <a href="mailto:rhiea@nih.gov">Arang Rhie</a>
</ul>
<h2>Credits</h2>
<p>Please cite Mc Cartney AM, Shafin K, Alonge M et al. <a href="https://doi.org/10.1038/s41592-022-01440-3">Chasing perfection: validation and polishing strategies for telomere-to-telomere genome assemblies.</a> Nat Methods (2022) doi: https://doi.org/10.1038/s41592-022-01440-3</p>

