<h2>Description</h2>
<p>Mappability from unique (single-copy) k-mers in the CHM13 assembly.</p>

<h2>Methods</h2>
<p>Unique k-mers were generated by first creating an initial Illumina marker database produced with meryl (v1.0~v1.3) from Illumina PCRfree CHM13 reads obtained from the SRA (PRJNA269593). A k=21 and k=51 sizes were selected for database construction and kmers within single-copy multiplicity (greater than 42 and less than 133 for k=21, greater than 28 and less than 107 for k=51) were retained. Kmers outside of these ranges were considered erroneous or multi-copy. Utilising this initial singles database, assembly specific single copy marker databases were constructed by intersected unique kmers found in the assembly.</p>

<h2>Track Descriptions</h2>
<ol>
	<li>Density: Density of the overlapping marker kmers, with the maximum overlapping number of marker kmers being the size of the k. The more dense the markers are present, the region is highly unique and easily mappable.</li>
	<li>Marker Desert: Regions at least 50kbp away from the nearest marker.</li>
</ol>

<h2>Data Access</h2>
Meryl marker databases are available to download <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/alignments/marker/">here.</a>

<h2>Release history</h2>
<ol>
  <li>v1.0 </li>
  <li>v1.1 </li>
</ol>

<h2>Contacts</h2>
<ul>
  <li><a href="mailto:ann.mccartney2@mail.dcu.ie">Ann Mc Cartney</a>, <a href="mailto:rhiea@nih.gov">Arang Rhie</a>
</ul>
<h2>Credits</h2>
<p>Please cite Mc Cartney AM, Shafin K, Alonge M et al. <a href="https://doi.org/10.1101/2021.07.02.450803">Chasing perfection: validation and polishing strategies for telomere-to-telomere genome assemblies.</a> bioRxiv (2021) doi: https://doi.org/10.1101/2021.07.02.450803</p>

